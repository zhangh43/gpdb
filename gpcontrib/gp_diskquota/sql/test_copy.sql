-- Test copy
create schema s3;
select diskquota.set_schema_quota('s3', '1 MB');
set search_path to s3;

create table c (i int);
copy c from '/tmp/csmall.txt';
-- expect failed 
insert into c select generate_series(1,100000000);
select pg_sleep(5);
-- select pg_total_table_size('c');
-- expect copy fail
copy c from '/tmp/csmall.txt';

drop table c;
reset search_path;
drop schema s3;
