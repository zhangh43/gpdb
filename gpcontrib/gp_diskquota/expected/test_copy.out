-- Test copy
create schema s3;
select diskquota.set_schema_quota('s3', '1 MB');
 set_schema_quota 
------------------
 
(1 row)

set search_path to s3;
create table c (i int);
copy c from '/tmp/csmall.txt';
-- expect failed 
insert into c select generate_series(1,100000000);
ERROR:  schema's disk space quota exceeded with name:s3
select pg_sleep(5);
 pg_sleep 
----------
 
(1 row)

-- select pg_total_table_size('c');
-- expect copy fail
copy c from '/tmp/csmall.txt';
ERROR:  schema's disk space quota exceeded with name:s3
drop table c;
reset search_path;
drop schema s3;
